### PR TITLE
Add a bundle test for a dsse envelope containing an in-toto statement

### PR DESCRIPTION
**Pull request by segiddins**
#### Summary
There are no bundle tests where `"kindVersion":{"kind":"dsse","version":"0.0.1"}` and the dsse envelope payload is an in-toto statement

Based on the example in https://blog.sigstore.dev/cosign-verify-bundles/

### CODE REVIEW

**Review by woodruffw — COMMENTED** [review]
LGTM behaviorally, although I think we might want to rename the bundle here (if only because naming a file `sha256:...` will make it harder to visually distinguish between file inputs and hash-literal inputs).

**Comment by segiddins** [comment]
Any suggested names? I picked this since it's what `gh attestation` created

**Comment by woodruffw** [comment]
> Any suggested names? I picked this since it's what `gh attestation` created

Hmm, maybe `intoto-in-dsse-v3.sigstore.json`? I admit that's not fantastic though.

**Comment by woodruffw** [comment]
I'll take this over and land it.

File: test/test_bundle.py [hunk 1]
```diff
@@ -452,8 +455,7 @@ def test_verify_in_toto_in_dsse_envelope(
             "https://github.com/cli/cli/.github/workflows/deployment.yml@refs/heads/trunk",
             "--certificate-oidc-issuer",
             "https://token.actions.githubusercontent.com",
-            "--verify-digest",
             f"sha256:{sha256}",
```
Comment (loosebazooka) [inline test/test_bundle.py:458]: I have some dsse bundles generated for `a.txt`. I can add those modifications in after though.

**Review by woodruffw — DISMISSED** [review]

**Review by loosebazooka — DISMISSED** [review]

**Review by woodruffw — APPROVED** [review]

**Pull request merged**